styles.css

```
@@ -8,24 +10,518 @@
+
 /* --------------- 4.2 Introduction ------------------- */
 
-/* ------------------------------- 5. Media Queries -------------------------------- */
\ No newline at end of file
+.section1 {
+    padding: 50px 0 50px 0;
+}
+
+.section1 p {
+    color: #4845E4;
+    padding: 10px 0 10px 0;
+    font-weight: 500;
+    font-size: 23px;
+    font-family: 'Merriweather', serif;
+}
+
+#introduction-question {
+    color: black;
+    padding: 50px 0 0 0;
+    font-size: 14px;
+    font-weight: 300;
+}
+
+#introduction-question a {
+    text-decoration: none;
+}
+
+/* ------------------------------- 4.3 Projects -------------------------------- */
+
+.section-article {
+    background-color: #F3F6F9;
+    padding: 100px 0 100px 0;
+}
+
+@media screen and (min-width: 576px) {
+    .section-article {
+        background-color: #F3F6F9;
+        padding: 10px 0 100px 0;
+    }
+}
+
+.article-projects {
+    margin: 0 0;
+    display: flex;
+    flex-direction: column-reverse;
+}
+
+@media screen and (min-width: 576px) {
+    .article-projects {
+        margin: 40px 0 40px 0;
+        display: flex;
+        flex-direction: row;
+    }
+}
+
+@media screen and (min-width: 576px) {
+    .article-projects:nth-child(even) {
+        display: flex;
+        flex-direction: row-reverse;
+    }
+}
+
+@media screen and (min-width: 576px) {
+    .article-projects:nth-child(even) .article-div {
+        /*border: 2px dashed pink;*/
+        margin: 10px 20px 10px 100px;
+    }
+}
+
+.article-div {
+    max-width: 100%;
+    margin: 30px 10px 30px 10px;
+
+}
+
+@media screen and (min-width: 576px) {
+    .article-div {
+        margin: 10px 100px 10px 20px;
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        max-width: 33%;
+        flex-grow: 0;
+        flex-shrink: 1;
+    }
+}
+
+.article-div img {
+    width: 40px;
+    padding: 30px 0 30px 0;
+}
+
+.article-div p {
+    padding: 30px 0 30px 0;
+
+    font-weight: 300;
+    font-size: 16px;
+}
+
+.article-div a {
+    padding: 30px 0 0 0;
+    text-decoration: none;
+    text-transform: uppercase;
+    font-size: 10px;
+    color: #5722CD;
+    font-weight: 700;
+    letter-spacing: 2px;
+    flex-shrink: 1;
+}
+
+.article-image {
+    width: 100%;
+    box-shadow: 4px 15px 40px grey;
+    flex-basis: 100%;
+}
+
+
+@media screen and (min-width: 576px) {
+    .image-wrapper {
+        flex-basis: 100%;
+        max-width: 67%;
+        flex-grow: 0;
+        flex-shrink: 1;
+    }
+}
+
+
+/* ------------------------------- 4.4 Services -------------------------------- */
+
+.services {
+    padding: 90px 0 90px 0;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    flex-wrap: wrap;
+    gap: 30px;
+}
+
+.services article {
+    flex-basis: 325px;
+    min-height: 325px;
+    min-width: 325px;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    border: 1px solid #4845E4;
+    padding: 10px 10px 10px 10px;
```

Hier kun je dan 
```suggestion
    padding: 10px;
```
van maken